index.js

```diff
@@ -237,6 +237,28 @@ const execa = (command, args, options) => {
 		return Promise.reject(error);
 	}
 
+	const originalKill = spawned.kill.bind(spawned);
+	spawned.kill = (signal, options) => {
+		if (signal === undefined || signal === 'SIGTERM') {
```

Comment:
`signal` should be case-insensitive.

Reply:
Yup, done: 8f590cb48d61837c1ba4572f743a92decdfa4c01